themes/beaver/assets/css/utilities/fl-builder-grid.css

```
@@ -0,0 +1,267 @@
+  z-index: 0;
+}
+
+.fl-row-bg-overlay .fl-row-content {
+  position: relative;
+  z-index: 1;
+}
+
+/* FL-Builder Row Height and Alignment */
+.fl-row-default-height .fl-row-content-wrap, .fl-row-custom-height .fl-row-content-wrap {
+  display: -webkit-box;
+  display: -webkit-flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)